apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -50,77 +51,134 @@ export function InstructorCourseAdminInstances({
+                              </td>
+                              <td class="align-left">${row.short_name}</td>
+                              <td class="align-left">${row.formatted_start_date}</td>
+                              <td class="align-left">${row.formatted_end_date}</td>
+                              <td class="align-middle">${row.enrollment_count}</td>
+                            </tr>
+                          `;
+                        })}
+                      </tbody>
+                    </table>
+                  </div>
+                `
+              : html`
+                  <div class="my-4 card-body text-center" style="text-wrap: balance;">
+                    <p class="font-weight-bold">No course instances found.</p>
+                    <p class="mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
+                    <p>
+                      Learn more in the
+                      <a
+                        href="https://prairielearn.readthedocs.io/en/latest/courseInstance/"
+                        target="_blank"
+                        >course instance documentation</a
+                      >.
+                    </p>
+                    ${resLocals.authz_data.has_course_permission_edit &&
+                    !resLocals.course.example_course &&
+                    !resLocals.needToSync
+                      ? html`
+                          <form class="ml-auto" name="add-course-instance-form" method="POST">
+                            <input
+                              type="hidden"
+                              name="__csrf_token"
+                              value="${resLocals.__csrf_token}"
+                            />
+                            <button
+                              name="__action"
+                              value="add_course_instance"
+                              class="btn btn-sm btn-primary"
+                            >
+                              <i class="fa fa-plus" aria-hidden="true"></i>
+                              <span>Add course instance</span>
+                            </button>
+                          </form>
+                        `
+                      : ''}
+                    ${!resLocals.authz_data.has_course_permission_edit
+                      ? html` <p>Course Editors can create new course instances.</p> `
+                      : ''}
+                    ${resLocals.course.example_course
+                      ? html` <p>You can't add course instances to the Example Course.</p> `
+                      : ''}
+                    ${resLocals.needToSync
+                      ? html`
+                          <p>
+                            Sync the course from the <strong>Sync</strong> page before creating a
+                            new course instance.
```

Comment:
It could be nice to link to the sync page here; something like "You must [sync the course](https://google.com) before creating a new course instance."

Reply:
I added this change:

<img width="827" alt="image" src="https://github.com/user-attachments/assets/555eb064-d07a-47d2-b6c0-2993da0dab80" />